### DISCUSSION TITLE
Google Search Results

### DISCUSSION THREAD

**Discussion by chrpai** [microsoft/fluentui-blazor#1477] [General]
I came across this in my search results. My apps are internal corporate apps but I was wondering if there's anyway to fix it and to make sure that whatever it is doesn't exist in the templates.

<img width="630" alt="image" src="https://github.com/microsoft/fluentui-blazor/assets/43178814/8069234a-7e62-4ba2-ab80-8bd2ff7788cd">

**Comment by NickHirras** [comment]
I don't understand your question. Where did you search, and where did you see the unexpected result?  @chrpai

**Reply by chrpai** [reply]
As the subject says, I searched Google for "fluentui blazor" and the image above was the second result.

In case your wondering. 'so what'...  SSR apps get a bad rap for SEO. and I just was hoping the root cause could be identified and any needed remediations are included in the project templates.

Does that make sense now?

**Reply by NickHirras** [reply]
Oh yes that makes sense. I don't think it's a fluent-blazor specific problem, but is being addressed upstream in the blazor framework itself. Blazor with DotNet 8 introduces some new render modes, including an "Auto" mode which uses server-side interactive rendering for initial page loads (which should be indexable by search engine crawlers), and then uses client-side rendering for subsequent requests after the Blazor bundle is downloaded.

You might be interested in these links:

https://www.youtube.com/watch?v=u4azTLLGt8U

https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0

**Reply by chrpai** [reply]
That's why I posted a discussion not an issue. I just wanted to raise awareness of what I saw when I searched.
